### PR TITLE
Check .Present before setting label value

### PR DESCRIPTION
**Pull request by jjzeng-seattle**
We shouldn't set the labels if the  value is not present.

This is necessary for the current knative work with multiple meters.  In knative some metric labels are extracted into resource labels, so we shouldn't set those metric labels.

https://github.com/evankanderson/knative-pkg/blob/resourcefullest/metrics/stackdriver_exporter.go#L239

### CODE REVIEW

File: metrics_test.go [hunk 1]
```diff
@@ -606,10 +606,12 @@ func TestResourceByDescriptor(t *testing.T) {
 						},
 						LabelValues: []metricdata.LabelValue{
 							{
-								Value: "v11",
+								Present: true,
```
Comment (nilebox) [inline metrics_test.go:609]: Could you please add a unit test for `Present: false`?

Comment (jjzeng-seattle) [reply metrics_test.go:609]: Done.

**Comment by nilebox** [comment]
@james-bebbington could you please take care of this PR if you have time?
Otherwise I'll get back to it next Tuesday.

**Pull request merged**